.vsts.pipelines/jobs/ci-local.yml

```diff
@@ -275,9 +275,20 @@ jobs:
     condition: eq(variables['sb.tarball'], true)
     continueOnError: true
 
+  # Most of these are directories we manually created and it's our responsibility to clean up (or the machine will fill up).
+  # /tmp/PackageVersions.props is created under sudo and later builds want to write to it unelevated so we have to clean that as well.
   - bash: |
       set -ex
-      rm -rf "$(tarballDirectory)" "$(stagingDirectory)" "$(dropDirectory)" "$(logsDirectory)"
-    displayName: Cleanup extra staging directories
+      sudo rm -rf "$(tarballDirectory)" "$(stagingDirectory)" "$(dropDirectory)" "$(logsDirectory)" /tmp/PackageVersions.props
+    displayName: Cleanup extra directories and files
+    condition: always()
+    continueOnError: true
+
+  # These can get stuck in a bad state, especially if the same machine is used for public and then an internal build or
```

Comment:
`public and then an internal build`

By this you mean something like release/5.0 versus internal/release/5.0 right?  The machines are only used within the context of dnceng/internal right? not dnceng/public.  This comment should clarify this.

Reply:
Yes, clarified this in the comment.